src/components/folder/NavBox.tsx

```diff
@@ -0,0 +1,138 @@
+import React, { useEffect, useState } from "react";
+import { getFolderNavInfo } from "../../api/folder.js";
+import * as S from "./NavBox.styled";
+import useModal from "../../hooks/useModal.ts";
+import NavItem from "./NavItem";
+import add from "../../assets/icon/add.svg";
+import shareIcon from "../../assets/icon/share.svg";
+import penIcon from "../../assets/icon/pen.svg";
+import deleteIcon from "../../assets/icon/delete.svg";
+import { ModalFuncParam } from "../../model/Modal";
+
+interface FolderNav {
+  navId?: number;
+  onClickNavItem: (navId: number) => void;
+}
+
+interface INavItem {
+  id?: number;
+  created_at?: string;
+  name?: string;
+  user_id?: number;
+  favorite?: boolean;
+  link?: {
+    count: number;
+  };
+}
+
+const NavBox = ({ navId, onClickNavItem }: FolderNav) => {
+  const { openModal } = useModal();
+  const [navList, setNavList] = useState<INavItem[]>([]);
+  const [currentNav, setCurrentNav] = useState("전체");
```

Comment:
true 와 같이 초기값만으로 형식을 유추할 수 없는 경우에는 왠만하면 타입 적어주세요!
```suggestion
  const [currentNav, setCurrentNav] = useState<string>("전체");
```